src/Core/Domain/Product/Customization/ValueObject/CustomizationId.php

```diff
@@ -53,16 +51,4 @@ public function getValue(): int
     {
         return $this->customizationId;
     }
-
-    /**
-     * @param int $value
-     *
-     * @throws CustomizationConstraintException
-     */
-    private function assertValueIsPositive(int $value)
-    {
-        if (0 >= $value) {
-            throw new CustomizationConstraintException(sprintf('Customization id must be positive integer. "%s" given', $value), CustomizationConstraintException::INVALID_ID);
-        }
-    }
```

Comment:
You should probably call it in the constructor rather than delete it 🤔

Reply:
It's a behavior change. But yes :)